COPYRIGHT

```diff
@@ -15,12 +15,13 @@ along with GAP; if not, write to the Free Software Foundation, Inc.,
 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
 
-Copyright (C) 1988-2018 by its authors, which include:
+Copyright (C) 1988-2020 by its authors, which include:
 
 * Ákos Seress
 * Alex Wegner
 * Alexander Hulpke
 * Alexander Konovalov
+* Anna Sucker
```

Comment:
This is sorted alphabetically by surnames and then names, right? Then Anna should be below Alice.

Reply:
This list is simply sorted alphabetically, because I was too lazy and just let me editor sort the lines (beware of non-ASCII chars like umlauts, though).

So yeah, Anna should be after Alice (and also after Andreas and Andrew, but before Ansgar)